CHANGELOG.md

```diff
@@ -1,7 +1,75 @@
 Change Log
 ==========
 
-# Version v4.0.0
+# Version 4.0.1
+
+_2024-10-01_
+
+This release contains a handful of bug fixes and improvements. 
+
+## 🧩 Compiler Plugins additions (experimental)
```

Comment:
```suggestion
## ⚙️ Add `ApolloCompilerPlugin.schemaListener()`
```

More specific title + my mind allocated 🧩 for the IJ plugin. Maybe not a huge deal but 🤷